config/deploy/local/kustomization.yaml

```diff
@@ -14,6 +14,9 @@ patches:
       - op: add
         path: /spec/template/spec/containers/0/args/-
         value: --zap-log-level=debug
+      - op: add
+        path: /spec/template/spec/containers/0/args/-
+        value: --e2e-test-only-use-mock-health-transport
```

Comment:
This means that every time we use `local-deploy` or `local-setup` the controller will expect the map for healthchecks to be present. I.E. we will not be able to have probes use a real transport layer when doing a request. 
The only way to have a "real" IP/CNAME probed is to `make run` and have the controller running locally